packages/datadog_flutter_plugin/example/ios/Tests/DatadogRumPluginTests.swift

```diff
@@ -165,6 +165,16 @@ class DatadogRumPluginTests: XCTestCase {
         testContracts(contracts: contracts, plugin: plugin)
     }
 
+    func testRumConfiguration_WithAppHangThreshold_IsSetCorrectly() {
+        let encoded: [String: Any?] = [
+            "applicationId": "fake-application-id",
+            "appHangThreshold": 3.2
```

Comment:
Could we not use a random generator here?

Reply:
👍 Added foundation mocks.